sub-packages/bionemo-core/src/bionemo/core/data/load.py

```diff
@@ -165,6 +165,7 @@ def load(
 
     download = pooch.retrieve(
         url=str(url),
+        fname=f"{resource.sha256}-{filename}",
```

Comment:
And `filename` is unique per resource, regardless if its on PBSS or NGC -- IIRC it's the key for each resource, right?

Reply:
Filename doesn't have to be unique, but I think the sha256 is enough of a guarantee of uniqueness? The filename is really just there to give some additional context. The "filename" here comes from the pbss link, and has to be the same for both pbss and ngc sources

Reply:
Ah, maybe misunderstood your comment -- the filename has consistent between ngc and pbss, yes